airbyte-integrations/connectors/source-google-ads/metadata.yaml

```diff
@@ -39,6 +39,12 @@ data:
           API. Users should refresh the source schema and reset affected streams after
           upgrading to ensure uninterrupted syncs.
         upgradeDeadline: "2023-11-30"
+      3.0.0:
+        message:
+          This release upgrades the Google Ads API to version 15 which causes changes in several schemas.
+          Users should refresh the source schema and reset affected streams after
+          upgrading to ensure uninterrupted syncs.
+        upgradeDeadline: "2023-12-25"
```

Comment:
Google is deprecating v13 of the Google Ads API in January. This release upgrades the Google Ads API to the latest version (v15), which causes changes in several schemas. Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.

Reply:
Updated